rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/logsdb/10_settings.yml

```diff
@@ -518,6 +518,48 @@ routing path not allowed in logs mode:
   - match: { error.type: "illegal_argument_exception" }
   - match: { error.reason: "[index.routing_path] requires [index.mode=time_series]" }
 
+---
+routing path allowed in logs mode with routing on sort fields:
+  - requires:
+      cluster_features: [ "routing.logsb_route_on_sort_fields" ]
+      reason: error message changed
+
+  - do:
+      indices.create:
+        index: test
+        body:
+          settings:
+            index:
+              mode: logsdb
+              number_of_replicas: 0
+              number_of_shards: 2
+              routing_path: [ host.name, agent_id ]
```

Comment:
Shouldn't this fail? Since `agent_id` isn't part of the sort fields?

Reply:
It's not a data stream..

Reply:
Ok, I see. I missed that.